src/gui/editors/PianoRoll.cpp

```diff
@@ -4360,8 +4421,22 @@ Note * PianoRoll::noteUnderMouse()
 	return NULL;
 }
 
+void PianoRoll::changeSnapMode()
+{
+	//	gridNudge,
+	//	gridSnap,
+	//	gridFree - to be implemented
 
-
+	QString qs_snapMode = m_snapModel.currentText();
+	if (qs_snapMode == "Nudge")
```

Comment:
Translations may break this.

Reply:
Would using `m_snapModel.getValue()` be better ?
It returns 0 (Nudge) and 1 (Snap).
I am uneasy using 'magic numbers' in code. Maybe add an `enum` for these values?

Reply:
I'd make an enum and compare to that.

Reply:
There's already an enum being used for `m_gridMode`, can't we just reuse that? I think just like the enum, the ComboBox model will go from 0 up, then you can cast and compare them.